docs/content/setup.md

```diff
@@ -36,7 +36,7 @@ PostCSS runner that suits your workflow:
 
 ---
 
-## Wait? What were is previous cssnext interface?
+## Wait? What where is the previous cssnext interface?
```

Comment:
Definitely.
This should be:
`Wait, what? Where is the previous cssnext interface?`